.github/workflows/ci.yml

```diff
@@ -227,6 +227,7 @@ jobs:
       - name: Cache scala dependencies
         uses: coursier/cache-action@v6
       - name: Install libuv
+        if: ${{ matrix.platform == 'JS' }}
```

Comment:
This is misunderstanding. Please remove "Install libuv" from all steps. We don't need it anymore.

Reply:
Done!

Reply:
But the Interesting here is that before I tested locally yesterday, disabling it and faced test timeout issues. But here not